ynr/apps/elections/views.py

```diff
@@ -256,13 +256,16 @@ def form_valid(self, form):
                 # If we're locking this, then the suggested posts
                 # can be deleted
                 ballot.suggestedpostlock_set.all().delete()
+                ballot.mark_uncontested_winners(
+                    request=self.request, ip_address=get_client_ip(self.request)
```

Comment:
The comment below has also made me realise - here we also want to pass in the `log=True` argument - to make sure we create a logged action.

It is only when we use the method in the management command where we don't want to create a logged action.